content/posts/2018-08-15/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Create a Multiuser GraphQL CRUD(L) App in 10 minutes with the new AWS Amplify CLI (and, in a few extra minutes, add search capabilities with Elasticsearch)
```

Comment:
Can we change the title to:
Create a Multiuser GraphQL CRUD(React) App in 10 minutes with the new AWS Amplify CLI (with ElasticSearch)

Reply:
Title changed